src/generated/event-payloads.ts

```diff
@@ -4568,13 +4568,4 @@ export declare module EventPayloads {
     organization?: WebhookPayloadCheckRunOrganization;
     installation?: WebhookPayloadCheckRunInstallation;
   };
-
-  interface WebhookEvent<T> {
-    id: string;
-    name: string;
-    payload: T;
-    protocol?: "http" | "https";
-    host?: string;
-    url?: string;
```

Comment:
no idea where these properties came from. My guess would be Probot. But we definitely don't use these